pyartcd/pyartcd/locks.py

```diff
@@ -124,9 +124,12 @@ async def lock(self, resource, *args, **kwargs):
         return lock
 
     async def unlock(self, lock):
-        self.logger.info('Releasing lock "%s"', lock.resource)
-        await super().unlock(lock)
-        self.logger.info('Lock released')
+        try:
+            self.logger.info('Releasing lock "%s"', lock.resource)
+            await super().unlock(lock)
+            self.logger.info('Lock released')
+        except LockError:
+            self.logger.warning('Failed releasing lock %s', lock.resource)
```

Comment:
Let's log the error message from the exception, and raise again?

Reply:
The LockError doesn't quite carry any useful information, I've tried in the past but seemed unrelevant

Reply:
But why not raise again, when this happens, as it would previously

Reply:
Right, updated